openvino/CMakeLists.txt

```diff
@@ -14,7 +14,7 @@
 # limitations under the License.
 # ******************************************************************************
 
-cmake_minimum_required(VERSION 3.10)
+cmake_minimum_required(VERSION 3.13)
```

Comment:
Let's remove it for this file. It's enough to have only in root cmake for OpenVINO